Neos.Flow.Log/Classes/DefaultLogger.php

```diff
@@ -99,14 +135,20 @@ public function addBackend(Backend\BackendInterface $backend)
      * @return void
      * @throws NoSuchBackendException if the given backend is unknown to this logger
      * @api
+     * @throws Exception
      */
     public function removeBackend(Backend\BackendInterface $backend)
     {
+        if ($this->wasPsrLoggerInjected) {
+            throw new Exception('A PSR-3 logger was injected so removing backends is not possible. Create a new instance.', 1515343007859);
+        }
+
         if (!$this->backends->contains($backend)) {
             throw new NoSuchBackendException('Backend is unknown to this logger.', 1229430381);
         }
         $backend->close();
         $this->backends->detach($backend);
+        $this->psrLogger = null;
```

Comment:
When can this ever happen to be needed? Ah, `createDefaultPsrLogger()` sets a logger, but not the inject flag. Maybe add a comment here?